Headers/IOKit/bluetooth/IOBluetoothHIDDriver.h

```diff
@@ -1,26 +1,44 @@
 /*
-	File:		IOBluetoothHIDDriver.h
-	Contains:	Driver for generic Bluetooth HID devices.
-	Copyright:	(c) 2001-2008 by Apple, all rights reserved.
-*/
-
-#ifndef IOBLUETOOTHHIDDRIVER_H
-#define IOBLUETOOTHHIDDRIVER_H
+ * Copyright (c) 2021 Apple Inc. All rights reserved.
```

Comment:
Better leave the copyright unchanged, unless it was published somewhere else. Add yours if you feel like it.

Reply:
Please drop Apple copyrights from reverse-engineered headers.

Reply:
This is not reverse-engineered -- I just fixed the spacing. So I think the Apple copyright should be retained.